### PR TITLE
feat(lesson): add schema for lesson

### PR DESCRIPTION
**Pull request by aldoprogrammer**
## Related Issue(s)

[//]:
  #
  "[Cite any related issue(s) or feature request(s) from the repository's issue tracker.]"

Fixes # For #

## Proposed Changes

[//]: # '[List down the specific changes you have made in this pull request.]'

- [x]

## Checklist

- [x] I have followed the repository's contribution guidelines.
- [x] I have added necessary documentation or updated existing documentation.
- [x] My code follows the established code style and formatting of the project.
- [x] I have tested my changes and they function as expected.
- [ ] I have added suitable unit tests (if applicable).

## API Changes documentation

```shell

```

```shell

```

## To-Do

- [ ]

### CODE REVIEW

File: prisma/schema.prisma [hunk 1]
```diff
@@ -57,3 +57,29 @@ model Audience {
...
+  title          String
+  sections       Section[]
+}
+
+model Section {
+  id             String     @id @default(auto()) @map("_id") @db.ObjectId
+  title          String
+  courseId       String     @db.ObjectId
+  course         Lesson     @relation(fields: [courseId], references: [id])
+  topics         Topic[]
```
Comment (Nasfame) [inline prisma/schema.prisma:72]: Store a string of an array of topics is not a good reason for creating a nwe collection

Comment this now, and reference this to create graphql types. 

i think its best 

schema Json @db.JsonArray //

**Review by Nasfame — CHANGES_REQUESTED** [review]

File: prisma/schema.prisma (hunk 1 above)
Comment (aldoprogrammer) [reply prisma/schema.prisma:72]:
```
model Section {
  id             String     @id @default(auto()) @map("_id") @db.ObjectId
  title          String
  courseId       String     @db.ObjectId
  course         Lesson     @relation(fields: [courseId], references: [id])
  topics         Json[]     @db.JsonArray
  @@map("sections")
}
```
I make it something like this, but it gives me error `Native type JsonArray is not supported for mongodb connector` And for my last coding which I store it as a string I follow this [docs ](https://www.prisma.io/docs/concepts/components/prisma-client/working-with-fields/working-with-scalar-lists-arrays)from Prisma official website.

File: prisma/schema.prisma [hunk 2]
```diff
@@ -57,3 +57,29 @@ model Audience {
   // @@unique([ageStart, ageEnd, level])
   @@map("audiences")
 }
+
+model Lesson {
+  id             String     @id @default(auto()) @map("_id") @db.ObjectId
+  title          String
+  sections       Section[]
+}
```
[the remark this answers is not in the document]
Comment (Nasfame) [reply prisma/schema.prisma:67]: Made in manually

**Review by Nasfame — APPROVED** [review]

**Review by Nasfame — APPROVED** [review]

File: prisma/schema.prisma (hunk 1 above)
Comment (Nasfame) [reply prisma/schema.prisma]: done

File: .env.example
[the remark this answers is not in the document]
Comment (Nasfame) [reply .env.example:1]: done

**Pull request merged**